internal/peer/common/peerclient.go

```diff
@@ -24,6 +24,14 @@ type PeerClient struct {
 	CommonClient
 }
 
+func NewPeerClient(address, tlsRootCertFile string) (*PeerClient, error) {
+	if address != "" {
+		return NewPeerClientForAddress(address, tlsRootCertFile)
+	} else {
+		return NewPeerClientFromEnv()
+	}
```

Comment:
Nit: now that this is in its own function no need for the else.
```suggestion
	} 
	return NewPeerClientFromEnv()
```

Reply:
Done